css/theme.css

```diff
@@ -110,7 +110,7 @@ iframe {
   flex-shrink: 0;
   border-radius: 3px;
   line-height: 30px;
-  background-color: #EEEEEE;
+  background-color: rgba(0, 0, 0, 0.07);
```

Comment:
I'm all for consistency, and don't mind changing this, but curious if you feel it applies to other CSS functions like `linear-gradient`?